NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/Resources.resx

```diff
@@ -121,11 +121,15 @@
+src: "https://js.monitor.azure.com/scripts/b/ai.2.min.js", // The SDK URL Source
+// name: "appInsights", // Global SDK Instance name defaults to "appInsights" when not supplied
+// ld: 0, // Defines the load delay (in ms) before attempting to load the sdk. -1 = block page load and add to head. (default) = 0ms load after timeout,
```

Comment:
A couple of minor nit's
Lets remove the commented out lines

Suggestion: Uncomment and keep the crossOrigin one.

Reply:
Just to verify;

- Lines 126, 127, 128 - delete
- Lines 129 - uncomment
- Lines 130 - delete

is this correct?

Reply:
Correct 😄